portfolio/src/main/webapp/script.js

```diff
@@ -50,12 +50,14 @@ function addRandomFact() {
 /**
  * Fetches message from server and adds it to DOM
  */
-function getMessage() {
-  fetch('/data')
-  .then(response => response.text())
-  .then((message) => {
-    // innerHTML is safe in this case because message comes from a static 
-    // String in the server. 
-    document.getElementById('message-container').innerHTML = message;
-  });
-}
\ No newline at end of file
+fetch('/data')
+.then(response => response.json())
+.then((message) => {
+  console.log(message[0]);
+  console.log(message[1]);
+  console.log(message[2]);
+
+  // Would replace with a for loop if there were more comments to display
```

Comment:
Another option is Array.prototype.join, which probably captures your intent better (and will naturally work when the comments array is a different length) 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join